src/main/java/frc/robot/subsystems/DriveBase.java

```diff
@@ -1,18 +1,15 @@
 package frc.robot.subsystems;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import edu.wpi.first.wpilibj.Encoder;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj.drive.DifferentialDrive;
 import frc.robot.Constants;
 import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.RelativeEncoder;
-import com.revrobotics.SparkMaxRelativeEncoder;
-import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.revrobotics.CANSparkLowLevel.MotorType;
```

Comment:
REV renamed their `com.revrobotics.CANSparkMaxLowLevel` class to `com.revrobotics.CANSparkLowLevel` (https://codedocs.revrobotics.com/java/com/revrobotics/cansparkmaxlowlevel).